src/components/MiniPlayer.jsx

```diff
@@ -1,28 +1,36 @@
-import { useState, useRef } from "react";
+import { useRef } from "react";
 
 import CloseIcon from "@/assets/svgs/icon-close.svg?react";
 import PauseIcon from "@/assets/svgs/icon-mini-pause.svg?react";
 import PlayIcon from "@/assets/svgs/icon-mini-player.svg?react";
+import { SETTING_TYPES } from "@/constants/settingOptions";
+import { useChannelStore } from "@/store/useChannelStore";
+import { useUserStore } from "@/store/useUserStore";
 import controlStreamingPlayback from "@/utils/playControl";
 
-const MiniPlayer = ({ thumbnail, channelName, closePlayer }) => {
-  const [isPlaying, setIsPlaying] = useState(false);
+const MiniPlayer = ({ closePlayer }) => {
+  const { selectedChannelId, radioChannelList, isPlaying, setIsPlaying } =
+    useChannelStore();
+  const { settings } = useUserStore();
+  const isAdDetect = settings[SETTING_TYPES.AD_DETECT];
+
+  const channel = radioChannelList.find(
+    (channel) => channel.id === selectedChannelId
```

Comment:
```suggestion
    ({ id }) => id === selectedChannelId
```
구조 분해 할당으로 축약할 수 있을 것 같아요!

Reply:
좋은 피드백 감사해요!
구조 분해를 통해 `({ id }) => id === selectedChannelId`로 작성하면 코드가 더 간결해지네요!
수정하여 반영했습니다 :)